sympy/plotting/plot.py

```diff
@@ -181,7 +184,10 @@ def __init__(self, *args, **kwargs):
         # The backend type. On every show() a new backend instance is created
         # in self._backend which is tightly coupled to the Plot instance
         # (thanks to the parent attribute of the backend).
-        self.backend = DefaultBackend
+        if 'usebackend' in kwargs:
```

Comment:
although it looks like the functions will be removed later i thought it's better not to tamper with already existing code and just use a different name.